deployment/helm/resource-management-policies/balloons/templates/clusterrole.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nri-resource-policy
```

Distinct naming. Plus, I think the Helm-specific labels should be applied to every single object

```suggestion
  name: nri-resource-policy-balloons
  labels:
    {{- include "nri-plugins.labels" . | nindent 4 }}
```

... i.e. ditto all other yamls (both the naming and the labels)

---

done